src/textUtils.c

```diff
@@ -59,21 +59,23 @@ size_t str_formatAdaAmount(uint64_t amount, char* out, size_t outSize)
 	return rawSize + suffixLength;
 }
 
-static size_t stringifyUint64ToBufferReverse(uint64_t number, char* ptr, char* end)
+static size_t stringifyUint64ToBufferReverse(uint64_t number, char* currChar, char* end)
```

Comment:
As I think about it, we generally pass buffer pointer and buffer size as arguments, e.g. for `str_formatAdaAmount`. I would prefer if this function conformed to this convention.
It's true you need end ptr for WRITE_CHAR, but you can compute it locally from the size.